pkgs/desktops/gnome-2/default.nix

```diff
@@ -57,8 +57,6 @@ lib.makeScope pkgs.newScope (self: with self; {
 
 #### DESKTOP
 
-  gvfs = pkgs.gvfs.override { gnome = self; };
```

Comment:
`throw` for removed attribute?

Reply:
I doubt anyone installed this.

Reply:
Could still have been used in someone's expression. At least then it'd be a message that it's gone.

Reply:
Moved it to aliases section since it was identical to gvfs in the first place.